### PR TITLE
review wanted

### PR DESCRIPTION
**Pull request by kajsabootin**
Link to my deployed project:
https://kajsa-happythought.netlify.com/

My reflections on how this week's project turned out:
- I thought this was a fun project, but it also was really hard.

Things I'd like to have clarified or explained in more detail:
- I struggled so much with the function "post new message "and also to be able to "like new message". But I made it work so it updates automatically.

### CODE REVIEW

File: code/src/components/HappyForm.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react"
+import "./HappyForm.css"
+
+export const HappyForm = (props) => {
```
Comment (ericaglimsholt) [inline code/src/components/HappyForm.js]: Maybe "happy" is not necessary?
```suggestion
export const Form = (props) => {
```

File: code/src/components/HappyForm.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <input 
+          
+          className="two-hearts-button"
+          onClick={handleSubmit} 
+          disabled={message.length < 5 || message.length > 140 ? true : false}
+          type="button" 
+          value="❤️ Send Happy Thoughts! ❤️"  />
+
+			</form>
+		</section>
```
Comment (ericaglimsholt) [inline code/src/components/HappyForm.js]: Some weird indention going on here

File: code/src/components/HappyHeart.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        method: "POST",
+        body: "",
+        headers: { "Content-Type": "application/json" }
+      }).then(() => {
+        setLikes(likes + 1)
+        setHasLiked(true)
+      })
+    }
+
+return (
```
Comment (ericaglimsholt) [inline code/src/components/HappyHeart.js]: I think this return could need some work. Now it isn't so easy to read...

Comment (kajsabootin) [reply code/src/components/HappyHeart.js]: Looks this better?

**Pull request closed without merging**